pages/price-feeds/use-real-time-data/evm.mdx

```diff
@@ -70,21 +48,38 @@ import "@pythnetwork/pyth-sdk-solidity/PythStructs.sol";
 contract SomeContract {
   IPyth pyth;
 
+  /**
+   * THIS IS AN EXAMPLE CONTRACT. IT IS NOT INTENDED TO BE USED IN PRODUCTION.
+   * THIS CONTRACT IS FOR DEMONSTRATION PURPOSES ONLY.
+   */
+
+  /**
+      * Instantiate IPyth interface with the address of the Pyth contract.
+      * Pyth contract address can be found at https://docs.pyth.network/price-feeds/contract-addresses/evm.
+
+      * @param pythContract The address of the Pyth contract
+      */
```

Comment:
I think we should document the actual relevant code (in this case, the line inside the constructor `pyth = IPyth(address)`, not the method. The method isn't the thing they're going to copy out of this.

I also think the comment should say something like

`The IPyth interface from pyth-sdk-solidity lets you access Pyth price feeds. Instantiate it with the Pyth contract address from <url>`